libmicrokit/Makefile

```diff
@@ -17,19 +17,23 @@ CFLAGS := -std=gnu11 -g3 -O3 -nostdlib -ffreestanding -mcpu=$(GCC_CPU) -Wall -Wn
 LIBS := libmicrokit.a
 OBJS := main.o dbg.o
 OTHER_OBJS :=  crt0.o
+ALLS := main_all.c
```

Comment:
```suggestion
# For verification, we concatenate all the C source files.
ALLS := microkit_all.c
```

We should probably follow the same style as the kernel with `*_all` and `*_all_pp` etc.

Reply:
Makes sense